Integrations/integration-IntSights.yml

```diff
@@ -1354,6 +1403,30 @@ script:
     - contextPath: IntSights.Alerts.IsFlagged
       description: Whether or not the alert is flagged
       type: boolean
+    - contextPath: IntSights.Alerts.Tags.CreatedBy
+      description: Name of the service the tag was created by
+      type: string
+    - contextPath: IntSights.Alerts.Tags.Name
+      description: Name of the tag
+      type: string
+    - contextPath: IntSights.Alerts.Tags._id
+      description: The ID of the tag
+      type: string
+    - contextPath: IntSights.Alerts.Images
+      description: The id of each image
+      type: string
+    - contextPath: IntSights.Alerts.Description
+      description: The description of the alert
+      type: string
+    - contextPath: IntSights.Alerts.Title
+      description: The title of the alert
+      type: string
+    - contextPath: IntSights.Alerts.TakedownStatus
+      description: The TakedownStatus of the alert
```

Comment:
Can we elaborate in the description? like possible takedown statuses?

Reply:
Not sure which ones are available, their documentation is does not specify that